pkg/processor/trigger/cron/trigger.go

```diff
@@ -189,6 +185,17 @@ func (c *cron) getMissedTicks(schedule cronlib.Schedule, nextEventSubmitTime tim
 	return missedTicks
 }
 
+func (c *cron) calculateNextEventSubmittingTime(lastEventSubmitTime time.Time) time.Time {
+	switch c.tickMethod {
+	case tickMethodSchedule:
+		return c.schedule.Next(lastEventSubmitTime)
+	case tickMethodInterval:
+		delay := c.schedule.(cronlib.ConstantDelaySchedule).Delay
+		return lastEventSubmitTime.Add(delay)
+	}
+	return time.Now()
```

Comment:
can use `default:` here instead of just `return time.now()`, to make it more readable